src/acl/agent.ts

```diff
@@ -37,59 +37,59 @@ import {
   internal_getAccessModes,
   internal_combineAccessModes,
   unstable_hasResourceAcl,
-  unstable_getResourceAcl,
   unstable_hasFallbackAcl,
-  unstable_getFallbackAcl,
 } from "../acl";
 
 export type unstable_AgentAccess = Record<WebId, unstable_AccessModes>;
 
 /**
- * Find out what Access Modes have been granted to a given Agent specifically for a given LitDataset.
+ * Find out what Access Modes have been granted to a given Agent specifically for a given Resource.
  *
  * Keep in mind that this function will not tell you what access the given Agent has through other ACL rules, e.g. public or group-specific permissions.
  *
  * Also, please note that this function is still experimental: its API can change in non-major releases.
  *
- * @param dataset The LitDataset to which the given Agent may have been granted access.
+ * @param resourceInfo Informations about the Resource to which Agents may have been granted access.
  * @param agent WebID of the Agent for which to retrieve what access it has to the Resource.
  * @returns Which Access Modes have been granted to the Agent specifically for the given LitDataset, or `null` if it could not be determined (e.g. because the current user does not have Control Access to a given Resource or its Container).
  */
 export function unstable_getAgentAccessModesOne(
-  dataset: LitDataset & WithResourceInfo & unstable_Acl,
+  resourceInfo: unstable_WithAcl["acl"],
   agent: WebId
 ): unstable_AccessModes | null {
-  if (unstable_hasResourceAcl(dataset)) {
-    const resourceAcl = unstable_getResourceAcl(dataset);
-    return unstable_getAgentResourceAccessModesOne(resourceAcl, agent);
+  if (unstable_hasResourceAcl(resourceInfo)) {
+    return unstable_getAgentResourceAccessModesOne(
+      resourceInfo.resourceAcl,
+      agent
+    );
   }
-  if (unstable_hasFallbackAcl(dataset)) {
-    const fallbackAcl = unstable_getFallbackAcl(dataset);
-    return unstable_getAgentDefaultAccessModesOne(fallbackAcl, agent);
+  if (unstable_hasFallbackAcl(resourceInfo)) {
+    return unstable_getAgentDefaultAccessModesOne(
+      resourceInfo.fallbackAcl,
+      agent
+    );
   }
   return null;
 }
 
 /**
- * Find out what Access Modes have been granted to specific Agents for a given LitDataset.
+ * Find out what Access Modes have been granted to specific Agents for a given Resource.
  *
  * Keep in mind that this function will not tell you what access arbitrary Agents might have through other ACL rules, e.g. public or group-specific permissions.
  *
  * Also, please note that this function is still experimental: its API can change in non-major releases.
  *
- * @param dataset The LitDataset to which Agents may have been granted access.
+ * @param resourceInfo Informations about the Resource to which Agents may have been granted access.
```

Comment:
```suggestion
 * @param resourceInfo Information about the Resource to which Agents may have been granted access.
```

Is "information" countable in French? I think I've seen this more often with French speakers :P

Reply:
It is, actually both "information" and "data" are, so if I'm not extra careful this happens stuck_out_tongue

Reply:
Yup, both 'information' and 'data' are countable in French, so if I'm not careful this happens ^^

Reply:
Really, "data" is countable? I mean, in Dutch it can serve as the plural version of "date", in which case it _is_ countable, but for the concept of data that is really weird. I wonder what I'd get if I asked a French person for one information or one data :P